libraries/provision/ansible/playbooks/tasks/install-sg-accel.yml

```diff
@@ -4,9 +4,9 @@
   shell: rpm -i /tmp/{{ couchbase_sg_accel_package }}
   when: ansible_distribution == "CentOS" or ansible_distribution == "RedHat"
 
-- name: SG ACCEL | Install sg_accel on Ubuntu
+- name: SG ACCEL | Install sg_accel rpm
```

Comment:
the package extension is deb, so change the rpm to deb

Reply:
done